# read the angle of rotating table

---
float64 current_pos
